packages/flutter/test/widget/animated_cross_fade_test.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+void main() {
+  testWidgets('AnimatedCrossFade test', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      new Center(
+        child: new AnimatedCrossFade(
+          firstChild: new SizedBox(
+            width: 100.0,
+            height: 100.0
+          ),
+          secondChild: new SizedBox(
+            width: 200.0,
+            height: 200.0,
+            child: new Text('visible')
+          ),
+          duration: const Duration(milliseconds: 200),
+          crossFadeState: CrossFadeState.showFirst
+        )
+      )
+    );
+
+    expect(find.byType(FadeTransition), findsNWidgets(2));
+    RenderBox box = tester.renderObject(find.byType(AnimatedCrossFade));
+    expect(box.size.width, equals(100.0));
```

Comment:
Shouldn't the cross fade widget be 200x200 when the animation starts and 100x100 when it completes?